web/src/stream_list_sort.ts

```diff
@@ -58,6 +56,10 @@ export function is_filtering_inactives(): boolean {
 }
 
 export function has_recent_activity(sub: StreamSubscription): boolean {
+    // TODO(evy) this is temporarily here for UI testing
+    if (sub.name === "design") {
+        return false;
+    }
```

Comment:
put this in its own commit, and also make some channels with inactive in the name and change to that